src/zjs_i2c_handler.c

```diff
@@ -4,7 +4,12 @@
 
 #define I2C_0 0
 #define I2C_1 1
+
+#ifdef CONFIG_BOARD_ARDUINO_101_SSS
+#define MAX_I2C_BUS 1
+#else
 #define MAX_I2C_BUS 2
+#endif
```

Comment:
@jimmy-huang I talked with @grgustaf about this, we determined it was better for the user experience to explicitly not allow it since it could lead to a headache on their end.  It literally can't do anything, so no point in letting them connect to it.  Should someone really want to, we can revisit that later :)